### PR TITLE
revert to original include guards

### PR DESCRIPTION
**Pull request by hanswenzel**
This release is geared to make larg4 part of CI to protect it from breaking changes. 

This release contains the following changes:
new larg4/Core/larg4SingleGen_module.cc: detector agnostic single particle generator
larg4/Core/CMakeLists: include larg4SingleGen_module

fcl files have been repaired to use new ParticleListAction defaults and to use larg4SingleGen
modified:   fcl/testlarg4.fcl
modified:   fcl/testmctruth.fcl

larg4/Services/LArG4Detector_service.cc : set xml schema evaluation to false

revert use of pragma once in header files

### CODE REVIEW

File: larg4/Services/SimEnergyDepositSD.h [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-//
-// artg4tk: art based Geant 4 Toolkit
 //
 //=============================================================================
 // SimEnergyDepositSD: Class representing a liquid Ar TPC
 // Author: Hans Wenzel (Fermilab)
 //=============================================================================
 
-#include "Geant4/G4VSensitiveDetector.hh"
-#include "lardataobj/Simulation/SimEnergyDeposit.h"
```
Comment (knoepfel) [inline larg4/Services/SimEnergyDepositSD.h]: Do you not need these header includes?

**Comment by knoepfel** [comment]
FYI, if you need to adjust the PR, just make the commit and push the commit to the same branch you used to create the PR.  No need to create a separate PR.

**Comment by lgarren** [comment]
@hanswenzel a local test build of this PR fails.  The failures indicate that headers are missing. See Kyle's comment.

**Comment by knoepfel** [comment]
trigger build

**Comment by absolution1** [comment]
approve

**Comment by lgarren** [comment]
approve

**Pull request merged**